src/front/app/components/challengeTopics/ChallengeTopics.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ChallengeTopic from './ChallengeTopic';
+
+let ChallengeTopics = ({challengeTopics = [], getChallengeTopics}) => {
+    if(!challengeTopics.length) {
+        getChallengeTopics();
+    }
+
+    return (
+        <ul>
+            {challengeTopics.map((challengeTopic, index) => {
+                return (
+                    < ChallengeTopic key={index} challengeTopic={challengeTopic}/>
```

Comment:
probably we should make eslint rules as step in build flow and don't push developers manually?